app/ffi/api/nfs.js

```diff
@@ -407,17 +407,25 @@ class NFS extends FfiApi {
       return error('Invalid parameters');
     }
     const self = this;
-    const fileMetadataPointerHandle = ref.alloc(PointerToFileMetadataPointer);
+    const fileMetadataHandle = ref.alloc(FileMetadataHandle);
     const executor = (resolve, reject) => {
       const onResult = (err, res) => {
         if (err || res !== 0) {
           return reject(err || res);
         }
         try {
-          const fileMetadataHandle = fileMetadataPointerHandle.deref();
-          const fileMetadataRef = ref.alloc(FileMetadataHandle, fileMetadataHandle).deref();
-          const metadata = derefFileMetadataStruct(fileMetadataRef.deref());
-          self.safeCore.file_metadata_drop.async(fileMetadataHandle, (e) => {});
+          let metadata = new FileMetadata(fileMetadataHandle.deref());
```

Comment:
This was fixed several days ago. Manual and automated tests are passing too. PR was rebased against latest master. Lint is happy etc. It is ready to re-review/merge.